core/renderers/common/constants.js

```diff
@@ -515,6 +471,8 @@ Blockly.blockRendering.ConstantProvider = function() {
     PUZZLE: 1,
     NOTCH: 2
   };
+
+  this.setTextConstants_();
```

Comment:
Why not do this in ``init``?

Reply:
Not sure what the correct order for these calls is.   No other constants depend on this one, so it could go in init, but lots of things depend on the padding one so it has to be done much earlier.  I chose to leave it in the same place as the old code.